### PR TITLE
feat: cc-0095 update courts schema and migration file

### PR DESCRIPTION
**Pull request by Kornzk**
update schema and courts migrate data

### CODE REVIEW

**Review by tatoMa — APPROVED** [review]
LGTM

**Review by stevenliu986 — APPROVED** [review]
LGTM

**Review by GitHubEason — APPROVED** [review]
LGTM

File: migrations/20220704011115-courtsData.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ module.exports = {
         sideBorderWidth: 1000,
         lineBorderWidth: 200,
         description: "PRO Full Court",
+        createdAt: new Date(),
```
Comment (chui-mel) [inline migrations/20220704011115-courtsData.js:17]: is this migration file can be changed? Have you tried?

**Pull request closed without merging**